### ISSUE TITLE
D3DCompiler_47.dll

### ISSUE THREAD

**Issue by Alamer99** [PeterTh/gedosato#371]
so i've been having a really weird issue with gedosato, i used it before and it was working no problem at all, then i formatted my pc install all vc++, directx, .net 4.6.1, but everytime i run a game with gedosato activated it gives me error D3DCompiler_47.dll is missing then the game just runs normal if gedosato is not running at all!

**Comment by mustang911** [comment]
Yeah, I notice the same issue. For me it starts with update 0.21.2307 2016/06/07 and downsampling not working anymore. Worst thing is I simply don't know how to roll back GeDoSaTo(

**Comment by Fukuroul** [comment]
@mustang911 You can use git to checkout to one of previous commits. There is a folder named "pack" contains a binary that already built from the source code.
